### PR TITLE
feat: update portal disclaimers

### PR DESCRIPTION
**Pull request by LucaMilanese90**
See: 
- https://github.com/deltaDAO/mvg-portal/pull/176
- https://github.com/deltaDAO/mvg-portal/pull/178 
- https://github.com/deltaDAO/mvg-portal/pull/182
- https://github.com/deltaDAO/mvg-portal/pull/189

### CODE REVIEW

**Review by moritzkirstein — CHANGES_REQUESTED** [review]
issue(blocking): The asset details page is currently crashing: 
grafik

**Comment by LucaMilanese90** [comment]
There was a typo in the assetDisclaimer.json file, should be fine now

File: content/pages/publish/form-algorithm.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
       "name": "files",
       "label": "File",
       "placeholder": "e.g. https://file.com/file.json",
-      "help": "Please enter the URL to your algorithm file and click \"ADD FILE\" to validate the data. This URL will be stored encrypted after publishing. Some restrictions apply:\n\n- max. running time: 1 min.\n- [Writing Algorithms for Compute to Data](https://docs.oceanprotocol.com/tutorials/compute-to-data-algorithms/)",
+      "help": "Please enter the URL to your algorithm file and click \"ADD FILE\" to validate the data. This URL will be stored permanently encrypted on-chain on the Gaia-X testnet after publishing. Some restrictions apply:\n\n- max. running time: 1 min.\n- [Writing Algorithms for Compute to Data](https://docs.oceanprotocol.com/tutorials/compute-to-data-algorithms/)",
```
Comment (moritzkirstein) [inline content/pages/publish/form-algorithm.json:22]: issue(blocking): updated `files` instead of `description`

**Review by moritzkirstein — CHANGES_REQUESTED** [review]

File: content/pages/publish/form-algorithm.json (hunk 1 above)
Comment (LucaMilanese90) [reply content/pages/publish/form-algorithm.json:22]: I don't see the error in this one, isn't this the right label already?

**Review by moritzkirstein — APPROVED** [review]

**Pull request merged**